app/code/Magento/Cms/Controller/Index/Index.php

```diff
@@ -14,33 +23,46 @@ class Index extends \Magento\Framework\App\Action\Action
     protected $resultForwardFactory;
 
     /**
-     * @param \Magento\Framework\App\Action\Context $context
-     * @param \Magento\Framework\Controller\Result\ForwardFactory $resultForwardFactory
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * @var Page
+     */
+    protected $page;
+
+    /**
+     * @param Context $context
+     * @param ScopeConfigInterface $scopeConfig
+     * @param ForwardFactory $resultForwardFactory
+     * @param Page $page
      */
     public function __construct(
-        \Magento\Framework\App\Action\Context $context,
-        \Magento\Framework\Controller\Result\ForwardFactory $resultForwardFactory
+        Context $context,
+        ScopeConfigInterface $scopeConfig,
+        ForwardFactory $resultForwardFactory,
+        Page $page
```

Comment:
Please, inject new dependencies according to [Magento Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)

Reply:
Changed to backward compatible